VERSION

```diff
@@ -1 +1 @@
-0.27.1
+0.29.1
```

Comment:
I'm not sure about how we update these

Reply:
current release in progress is 0.29.0, so this seems right.